### ISSUE TITLE
PL install/generation fails after initial install if ignored files do not exist

### ISSUE THREAD

**Issue by coreylafferty** [pattern-lab/patternlab-php-core#41]
I'm trying to update a project to use 2.0. On install everything works fine, but not if ignored files are deleted.  For example, vendor/, public/ and source/styleguide/ are all in the .gitignore, so if I commit this project and reclone it (or someone else pulls it down), those files and directories won't exist. Running 'composer update' or 'composer install' (which, prior to the upgrade, would recreate the ignored files) shows this error: 
```
"hrm... i can't seem to find the directory where you want to write your styleguide. are you sure it's at ./source? you can fix this in ./config/config.yml by editing publicDir. sorry, stopping pattern lab... :("
```

**Comment by dmolsen** [comment]
@coreylafferty -

Two problems on my end. One is a bad assumption and the other is a bad `.gitignore`. The latter should probably be:

```
.DS_Store
export/*
public/*
!public/.gitkeep
source/styleguide/*
vendor/*
```

But I can default to building the directories instead of throwing errors since I've defaulted to that in the past. Thanks again.

**Comment by dmolsen** [comment]
The bad assumption has been addressed in `dev`. Working through some other issues that I'll bundle into one release. I'll close this when that's out.

**Comment by dmolsen** [comment]
Released with `v2.2.0`
